### ISSUE TITLE
Forums cancer

### ISSUE THREAD

**Issue by nolanw** [Awful/Awful.app#512]
It's a thing and we've never shown it like the website does.

**Comment by JonET** [comment]
It's a dumb thing and the app is better for not showing it.

**Comment by nolanw** [comment]
Agreed, but my general philosophy for the app is to stay true to the Forums. Which is corny af, but it means implementing all the dumb shit you see on the Forums, like cancer.

It’s pretty far down my list though, I’ll give you that!

**Comment by nolanw** [comment]
> I don't know if anyone cares, but forums cancer doesn't get rendered in the app.
> — [anthonypants](https://forums.somethingawful.com/showthread.php?threadid=3837546&userid=0&perpage=40&pagenumber=28#post489356229)
